website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,8 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## October 2024
 
+- **Behavior change:** Set [`state_modified_compare_more_unrendered`](/reference/global-configs/behavior-changes#source-definitions-for-state) to true to reduce false positives for `state:modified` when configs differ between `dev` and `prod` environments.
+- **Behavior change:** Set the [`skip_nodes_if_on_run_start_fails`](/reference/global-configs/behavior-changes#on-run-start-hook) flag to `True` to skip all selected resources from running if there is a failure on an `on-run-start` hook.
```

Comment:
If [other suggestion below](https://github.com/dbt-labs/docs.getdbt.com/pull/6204/files#r1786503120) is adopted, then will also need to do something like this:

```suggestion
- **Behavior change:** Set the [`skip_nodes_if_on_run_start_fails`](/reference/global-configs/behavior-changes#failures-in-on-run-start-hooks) flag to `True` to skip all selected resources from running if there is a failure on an `on-run-start` hook.
```